scripts/link-checker

```diff
@@ -42,6 +42,7 @@ END {
 
 WHITELIST=$(mktemp)
 grep -v '^#' "$SCRIPT_DIR/../tests/linkchecker.whitelist" > "$WHITELIST"
+sed -i '/^$/d' "$WHITELIST"
```

Comment:
Unfortunately [using `-i` without any additional argument will not work, if you use the BSD version of `sed`](https://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux). I think we can work around this problem using `grep` to drop both the empty lines and comments.

```suggestion
grep -Ev '^(#|[[:space:]]*$)' "$SCRIPT_DIR/../tests/linkchecker.whitelist" > "$WHITELIST"
```